airplanedev/cli
# Remove inline logic in views init

## Description

Removes the `--inline` options for views init and clean up the init code that switched on inline/noninline

## Test plan

* Build CLI locally and successfully views init
* Verify `--inline` option is not supported
* Tests pass
